library/src/levenshtein.cxx

```diff
@@ -0,0 +1,71 @@
+#include "levenshtein.h"
```

Comment:
do this require its own source file ? Why not put that in utils.cxx directly ?

Reply:
I'm expecting utils.cxx to become much bigger in the future so I think it's better to separate things.

Reply:
fair enough